bucket/oss-cad-suite-nightly.json

```diff
@@ -0,0 +1,37 @@
+{
+    "version": "2022-03-29",
+    "description": "Open source digital design and verification tools. Includes tools for RTL synthesis, formal hardware verification, place & route, FPGA programming, and testing with support for HDLs like Verilog, Migen and Amaranth.",
+    "homepage": "https://github.com/YosysHQ/oss-cad-suite-build",
+    "license": "ISC",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/YosysHQ/oss-cad-suite-build/releases/download/2022-03-29/oss-cad-suite-windows-x64-20220329.exe#/dl.7z",
+            "hash": "c2b10a627d960952a969c3cbdc9bf4109b7ee533c8b43a0c06df2c0cf3eacf45"
+        }
+    },
+    "extract_dir": "oss-cad-suite",
+    "pre_install": "Set-Content -Path \"$dir\\start.bat\" -Value \"@cmd /k $dir\\environment.bat\"",
+    "bin": [
+        [
+            "start.bat",
+            "oss-cad"
+        ]
+    ],
+    "shortcuts": [
+        [
+            "start.bat",
```

Comment:
Will this open a GUI?

Reply:
No, it just opens a command prompt window. I suppose it's not really necessary but maybe quicker than opening a terminal yourself and then running `oss-cad`?

Reply:
I don't remember if I've seen something like this - quite unconventional...

Reply:
I suppose it's a bit like the Git Bash shortcut that opens a self-contained command prompt window?

Reply:
The Git Bash shortcut is unique because it opens a GUI - the `mintty.exe` terminal, which is very different from a normal command prompt window. 

Also I just noticed that `start.bat` file already exists.

Reply:
Updated the manifest, now it's just a shim to load the environment and now scoop users can use the latest version of Yosys:
```
'oss-cad-suite-nightly' (2022-03-31) was installed successfully!

scoop\buckets\testing via 🐍 v3.10.2 took 2m25s
❯ oss-cad

[OSS CAD Suite] C:\Users\me\scoop\buckets\testing>yosys -V
Yosys 0.15+81 (git sha1 fc2af4e32, x86_64-w64-mingw32-g++ 9.2.1 -Os)
```